### PR TITLE
Underdog Media 1.0 Compliance

### PR DESCRIPTION
**Pull request by Jacobkmiller**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ x] Other

## Description of change

Update of Underdog Media adapter for 1.0 Compliance

### CODE REVIEW

File: modules/underdogmediaBidAdapter.js [hunk 1]
```diff
@@ -1,112 +1,116 @@
...
-      }
     });
-  }
 
-  $$PREBID_GLOBAL$$.handleUnderdogMediaCB = _callback;
+    return bidResponses;
+  },
 
-  function _makeNotification(bid, mid, bidParam) {
+  makeNotification: function (bid, mid, bidParam) {
```
Comment (jaiminpanchal27) [inline modules/underdogmediaBidAdapter.js]: `makeNotification` and `getUrlVars` function can remain private in adapter. No need to added them to `spec`

File: modules/underdogmediaBidAdapter.js [hunk 2]
```diff
@@ -1,112 +1,116 @@
...
 
-  function _callback(response) {
-    var mids = response.mids;
-    bidParams.bids.forEach(bidParam => {
-      var filled = false;
-      mids.forEach(mid => {
+    return {
+      method: 'GET',
+      url: `${window.location.protocol}//udmserve.net/udm/img.fetch`,
+      data: `tid=1;dt=10;sid=${siteId};sizes=${sizes.join(',')}`,
```
Comment (jaiminpanchal27) [inline modules/underdogmediaBidAdapter.js]: More params can be added to request object here. So you can add `bidParams` to bidRequest and access that in `interpretResponse` method. 
By doing this you can remove `bidParams` line 15.

File: test/spec/modules/underdogmediaBidAdapter_spec.js [hunk 3]
```diff
@@ -1,122 +1,241 @@
...
+        let bidRequests = [
+          {
+            bidId: '3c9408cdbf2f68',
+            sizes: [[300, 250]],
+            bidder: 'underdogmedia',
+            params: {
+              siteId: '12143'
+            },
+            requestId: '10b327aa396609',
+            placementCode: '/123456/header-bid-tag-1'
```
Comment (jaiminpanchal27) [inline test/spec/modules/underdogmediaBidAdapter_spec.js]: In 1.0 we have renamed `placementCode` to `adUnitCode`. Please rename

**Comment by Jacobkmiller** [comment]
Thanks @jaiminpanchal27 for the review. I have updated the PR with your changes.

**Pull request merged**